vivisect/base.py

```diff
@@ -669,19 +673,35 @@ def _initFunction(self, funcva):
     #def _loadImportApi(self, apidict):
         #self._imp_api.update( apidict )
 
+    def getEndian(self):
+        return self.bigend
+
+    def _doSetEndian(self, endian):
```

Comment:
Does anybody else call `_doSetEndian`?

Reply:
not currently.
would you like me to merge it ito `handleENDIAN()`?
i left them separate because we may want the ability to push endianness to all archs without making an event, but as i'm typing this up i'm realizing how bad an idea that sounds.

Reply:
done